docker-compose.yml

```diff
@@ -5,7 +5,8 @@ services:
     image: pcic/sandpiper
     ports:
       - "8101:5003"
-
+    volumes:
+      - /storage/data/climate/downscale/BCCAQ2/:/storage/data/climate/downscale/BCCAQ2/
```

Comment:
I wonder what this line is for. Would be appreciated if you could teach me.

Reply:
I discovered that our deployed instance of `sandpiper` was unable to correctly resolve the rules as it did not have access to the data. This just supplies the data as a volume that can be targeted using the same directory structure the database anticipates.